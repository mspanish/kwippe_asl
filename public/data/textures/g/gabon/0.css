/*
-------------------------------------------------------------------
Created with TexturePacker https://www.codeandweb.com/texturepacker
-------------------------------------------------------------------
Inline:     <i class="yourspritesname"></i>
Responsive: <div class="yourspritesname"></div>
*/


.@a16089@emo@ga {max-width:75px; max-height:75px; }
.@a16089@emo@ga::after {content: '\00a0';display: inline-block; width:75px; height:75px; background-position: 0.851063829787234% 50%;background-size: 413.3333333333333% 105.33333333333333%;background-image: url(./textures/g/gabon/0.png/0.png);padding: 0; }
div.@a16089@emo@ga::after {max-width:75px; width:100%;height:0;padding: 0 0 100% 0;}
.@a37041@mul@gabon {max-width:75px; max-height:75px; }
.@a37041@mul@gabon::after {content: '\00a0';display: inline-block; width:75px; height:75px; background-position: 33.61702127659574% 50%;background-size: 413.3333333333333% 105.33333333333333%;background-image: url(./textures/g/gabon/0.png/0.png);padding: 0; }
div.@a37041@mul@gabon::after {max-width:75px; width:100%;height:0;padding: 0 0 100% 0;}
.@d1973@wikim-all@coat_of_arms_of_gabon {max-width:75px; max-height:75px; }
.@d1973@wikim-all@coat_of_arms_of_gabon::after {content: '\00a0';display: inline-block; width:75px; height:75px; background-position: 66.38297872340425% 50%;background-size: 413.3333333333333% 105.33333333333333%;background-image: url(./textures/g/gabon/0.png/0.png);padding: 0; }
div.@d1973@wikim-all@coat_of_arms_of_gabon::after {max-width:75px; width:100%;height:0;padding: 0 0 100% 0;}
.@d2485@wikim-all@flag_of_gabon {max-width:75px; max-height:75px; }
.@d2485@wikim-all@flag_of_gabon::after {content: '\00a0';display: inline-block; width:75px; height:75px; background-position: 99.14893617021276% 50%;background-size: 413.3333333333333% 105.33333333333333%;background-image: url(./textures/g/gabon/0.png/0.png);padding: 0; }
div.@d2485@wikim-all@flag_of_gabon::after {max-width:75px; width:100%;height:0;padding: 0 0 100% 0;}

/* $TexturePacker:SmartUpdate:3c6b1d2647b73cd82681e0c23c7230db:3d48289081f8f97ac5289939ac9bbe85:4ddb42793a59c1b7bfff1539dfc4d068$ */
