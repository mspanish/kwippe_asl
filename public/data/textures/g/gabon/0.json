
{

"@a16089@emo@ga": { 
	"vb": "75 75 -0 -0",
 },
"@a37041@mul@gabon": { 
	"vb": "75 75 -0 -75",
 },
"@d1973@wikim-all@coat_of_arms_of_gabon": { 
	"vb": "75 75 -0 -150",
 },
"@d2485@wikim-all@flag_of_gabon": { 
	"vb": "75 59 -0 -225",
 },
}
