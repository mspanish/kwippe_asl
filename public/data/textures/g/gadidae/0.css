/*
-------------------------------------------------------------------
Created with TexturePacker https://www.codeandweb.com/texturepacker
-------------------------------------------------------------------
Inline:     <i class="yourspritesname"></i>
Responsive: <div class="yourspritesname"></div>
*/


.@a59377@ian@arctogadus-glacialis {max-width:75px; max-height:75px; }
.@a59377@ian@arctogadus-glacialis::after {content: '\00a0';display: inline-block; width:75px; height:75px; background-position: 2.4691358024691357% 50%;background-size: 208% 105.33333333333333%;background-image: url(./textures/g/gadidae/0.png/0.png);padding: 0; }
div.@a59377@ian@arctogadus-glacialis::after {max-width:75px; width:100%;height:0;padding: 0 0 100% 0;}
.@a60344@ian@gadus-morhua {max-width:75px; max-height:75px; }
.@a60344@ian@gadus-morhua::after {content: '\00a0';display: inline-block; width:75px; height:75px; background-position: 97.53086419753086% 50%;background-size: 208% 105.33333333333333%;background-image: url(./textures/g/gadidae/0.png/0.png);padding: 0; }
div.@a60344@ian@gadus-morhua::after {max-width:75px; width:100%;height:0;padding: 0 0 100% 0;}

/* $TexturePacker:SmartUpdate:c3945144a94e2f2251f8736eca06c3e3:2928536336c02a9574e6b17fa770e843:4ddb42793a59c1b7bfff1539dfc4d068$ */
