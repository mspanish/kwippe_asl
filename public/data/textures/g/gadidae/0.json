
{

"@a59377@ian@arctogadus-glacialis": { 
	"vb": "75 25 -0 -29",
 },
"@a60344@ian@gadus-morhua": { 
	"vb": "75 29 -0 -0",
 },
}
