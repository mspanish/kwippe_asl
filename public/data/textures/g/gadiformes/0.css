/*
-------------------------------------------------------------------
Created with TexturePacker https://www.codeandweb.com/texturepacker
-------------------------------------------------------------------
Inline:     <i class="yourspritesname"></i>
Responsive: <div class="yourspritesname"></div>
*/


.@a59377@ian@arctogadus-glacialis {max-width:75px; max-height:75px; }
.@a59377@ian@arctogadus-glacialis::after {content: '\00a0';display: inline-block; width:75px; height:75px; background-position: 1.2658227848101267% 50%;background-size: 310.6666666666667% 105.33333333333333%;background-image: url(./textures/g/gadiformes/0.png/0.png);padding: 0; }
div.@a59377@ian@arctogadus-glacialis::after {max-width:75px; width:100%;height:0;padding: 0 0 100% 0;}
.@a60344@ian@gadus-morhua {max-width:75px; max-height:75px; }
.@a60344@ian@gadus-morhua::after {content: '\00a0';display: inline-block; width:75px; height:75px; background-position: 50% 50%;background-size: 310.6666666666667% 105.33333333333333%;background-image: url(./textures/g/gadiformes/0.png/0.png);padding: 0; }
div.@a60344@ian@gadus-morhua::after {max-width:75px; width:100%;height:0;padding: 0 0 100% 0;}
.@a61930@ian@urophycis-regia {max-width:75px; max-height:75px; }
.@a61930@ian@urophycis-regia::after {content: '\00a0';display: inline-block; width:75px; height:75px; background-position: 98.73417721518987% 50%;background-size: 310.6666666666667% 105.33333333333333%;background-image: url(./textures/g/gadiformes/0.png/0.png);padding: 0; }
div.@a61930@ian@urophycis-regia::after {max-width:75px; width:100%;height:0;padding: 0 0 100% 0;}

/* $TexturePacker:SmartUpdate:567dabeb6007fc8104a0f4a029405829:e677f4ca6cf1b96307466f5b33670085:4ddb42793a59c1b7bfff1539dfc4d068$ */
