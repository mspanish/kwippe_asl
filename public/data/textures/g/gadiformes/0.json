
{

"@a59377@ian@arctogadus-glacialis": { 
	"vb": "75 25 -75 -0",
 },
"@a60344@ian@gadus-morhua": { 
	"vb": "75 29 -0 -0",
 },
"@a61930@ian@urophycis-regia": { 
	"vb": "75 25 -150 -0",
 },
}
