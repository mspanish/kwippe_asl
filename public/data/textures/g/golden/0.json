
{

"@a59600@ian@cantharellus-cibarius": { 
	"vb": "75 73 -0 -0",
 },
"@a59896@ian@corbicula-fluminea": { 
	"vb": "75 71 -0 -73",
 },
"@a60348@ian@galerucella-pusilla": { 
	"vb": "75 39 -0 -144",
 },
}
